src/sage/rings/laurent_series_ring.py

```diff
@@ -221,12 +221,14 @@ def __classcall__(cls, *args, **kwds):
             'q'
         """
         from .power_series_ring import PowerSeriesRing
-
+        if 'default_prec' in kwds and kwds['default_prec'] is infinity:
```

Comment:
```suggestion
        if kwds.get('default_prec', None) is infinity:
```
Personal style choice here.